app/lib/finance/api/response.rb

```diff
@@ -0,0 +1,6 @@
+module Finance
+  module Api
+    module Response
+    end
+  end
+end
```

Comment:
remember why do we need this?

Reply:
Thales did it [here](https://github.com/3scale/porta/pull/3339/commits/f15bfd6b604b480575456cadb0ae114d81a1bfd5#r1238505018).

I think this was a leftover or unused file, and I removed it in the later cleanup commit, [here](https://github.com/3scale/porta/pull/3456/commits/eb2346546cd7560f201510c26e8ec38b5fd52b6d#diff-2420eda950edd6e9d6e91336681bff689d2cd4a8d9d81118ab8ea3ac08ae2369)